README.md

```diff
@@ -6,27 +6,23 @@ Collection of utility methods for Android and WordPress.
 
 * In your build.gradle:
 ```groovy
-dependencies {
-    compile 'org.wordpress:utils:1.30.0' // use version 1.30.0
+repositories {
+    maven { url "https://a8c-libs.s3.amazonaws.com/android" }
 }
-```
-
-## Publish an updated version to your local maven repository
-
-You can bump the [version name in the main build file: `WordPressUtils/build.gradle`][1]. After updating the build file, you can build, and publish the library to your local maven repo. That will let you try the new version in your app for example.
 
-```shell
-$ ./gradlew assemble test publishToMavenLocal
+dependencies {
+    implementation 'org.wordpress:utils:1.31.0'
```

Comment:
This version doesn't exist yet. I think we should publish it once we merge S3 changes to `develop`.